Go/fibonacci/main.go

```diff
@@ -1,19 +1,32 @@
-// Simple Program to calculate fibonacci of input
-
 package main
 
-import "fmt"
-func FibonacciRecursion(n int) int {
+import (
+	"fmt"
+	"strconv"
+	"os"
+)
+
+func FibonacciSequence(n int) int {
     if n <= 1 {
         return n
     }
-    return FibonacciRecursion(n-1) + FibonacciRecursion(n-2)
+    return FibonacciSequence(n-1) + FibonacciSequence(n-2)
 }
 
 func main(){
-    fmt.Print("Enter number : ")
-    var n int
-    fmt.Scanln(&n)
-
-    fmt.Println("Fibonacci of", n , "is", FibonacciRecursion(n));
+    if len(os.Args) > 1 {
+    	for _, arg := range os.Args[1:] {
+    		n, _ := strconv.Atoi(arg)
```

Comment:
The error should be handled, preferably by printing to stderr, but just panic is good enough for now